packages/seo-integration/src/social/containers/facebook-container.js

```diff
@@ -37,8 +38,8 @@ const FacebookEditorContainer = ( { as: Component, ...restProps } ) => {
 		recommendedReplacementVariables={ recommendedReplacementVariables }
 		titleInputPlaceholder={ "" }
 		descriptionInputPlaceholder={ "" }
-		descriptionPreviewFallback={ socialDescriptionTemplate || contentDescription || contentExcerpt || "" }
-		titlePreviewFallback={ socialTitleTemplate || seoTitle || "" }
+		descriptionPreviewFallback={ socialDescriptionTemplate || contentDescription || description || contentExcerpt || "" }
+		titlePreviewFallback={ socialTitleTemplate || seoTitle || titleNoFallback || title || "" }
```

Comment:
See my other comment about moving the fallback logic to the selectors. It may make this a bit more readable and make the fallback logic agnostic 😄 .